standards/MEETUP_TALKS.md

```diff
@@ -0,0 +1,57 @@
+# How to organise a talk based meet up
+
+## Introduction
+
+The purpose of this document is to provide a checklist and timeline for organising talk based meetups. This is based around the process used by the [London Rust User Group][link_lrug].
+
+The expected audience for this document are people who are new to setting up meetups or organising this type of format.
+
+Where relevant I will cite actual examples using the names of services or organisations (Skillsmatter, meetup.com, etc) that we currently use at the London user group.
+
```

Comment:
Also ask them to fill in https://docs.google.com/forms/d/e/1FAIpQLSf52YXGhqBaHtCXtVna4iHYMK7IQaTqUW6V-ztsZC8C2TBInQ/viewform and join https://groups.google.com/forum/#!forum/rust-meetup-organizers  so they have a place to ask questions?